lib/Agrammon/Web/Service.pm6

```diff
@@ -38,29 +38,40 @@ class Agrammon::Web::Service {
         return Agrammon::DB::Datasets.new(:$user, :$version).load.list;
     }
 
+    method delete-datasets(Agrammon::Web::SessionUser $user, @datasets) {
+        return @datasets.elems;
+        # return Agrammon::DB::Datasets.new(:$user, :$version).load.list;
+    }
+
+    # return list of datasets as expected by Web GUI
+    method send-datasets(Agrammon::Web::SessionUser $user, @datasets) {
+        return @datasets.elems;
+#        return Agrammon::DB::Datasets.new(:$user, :$version).load.list;
+    }
```

Comment:
Left as a reminder, or leftover commented code to remove?

Reply:
One of the things to now get into sync between frontend and backend.